Jasmine/Tetris/ViewModels/TetrisViewModel.swift

```diff
@@ -1,3 +1,14 @@
+import Foundation
+
 class TetrisViewModel: TimedGameViewModel {
+    /// Specifies the total time allowed in the game.
+    var totalTimeAllowed: TimeInterval
+
+    /// Specifies the time remaining in the game.
+    var timeRemaining: TimeInterval
 
+    init() {
+        totalTimeAllowed = 0
+        timeRemaining = 0
+    }
```

Comment:
Right, I see now. I'm not really familiar on how to adopt the technique you proposed though, do you want to change it?